service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataService.java

```diff
@@ -212,12 +224,12 @@ private String deriveApplicationType(final Orders orders) {
 
         if (selectedOrderTypes.size() > 1) {
             return selectedOrderTypes.stream()
-                .map(this::getOrderTypeLabelValue)
+                .map(selectedOrderType -> capitalize(getOrderTypeLabelValue(selectedOrderType), EMPTY_CHAR))
```

Comment:
I would be cautious with using our labels capitalised, if business decided to amend our lables then automatically it will break robotics integration, I would prefer explicit mapping of our enums to required strings.
please confirm if Discharge of a Care Order should be all capitalised as currently story says 'of' should start with small letter but this implementation makes is capital